app/helpers/application_helper.rb

```diff
@@ -31,7 +31,13 @@ def time_ago_in_words(from_time, include_seconds=false)
   end
 
   def tab_item(label, path)
-    "<li class='#{request.path == path ? 'selected' : ''}'><a href='#{path}'>#{label}</a></li>"
+    if label === 'Home'
+      "<li class='#{request.path == path ? 'selected' : ''}'><a href='#{path}'>#{label}</a></li>"
+    elsif label === 'Visits'
+      "<li class='#{request.path.include?('/visits/') ? 'selected' : ''}'><a href='#{path}'>#{label}</a></li>"
+    else
+       "<li class='#{request.path.start_with?(path) ? 'selected' : ''}'><a href='#{path}'>#{label}</a></li>"
```

Comment:
It seems like this line or something similar should be all that is needed to replace the original `request.path == path` check. Do we need the three checks?

Reply:
if we only use `start_with`, 'Home' will also be selected when folks visit the Visits/Reports/Settings routes.

if I knew as much about ruby as I do about JS, I could have condensed the other two by separating the root of the default 'Visits' path from today's date, but I took the easy way out.

Reply:
this would _probably_ work.
```ruby
if label === 'Home'
    "<li class='#{request.path == path ? 'selected' : ''}'><a href='#{path}'>#{label}</a></li>"
else
    "<li class='#{request.path.include?('/#{Label.downcase}/') ? 'selected' : ''}'><a href='#{path}'>#{label}</a></li>"
```
I'll find some time to make sure soon.

Reply:
nope, AFAICT, the two pesky routes below make it pretty tough to cram all the logic you'd need in without a third clause.

* /[org]/reports
* /[/org]/reports/visits

if `/[org]/reports/` was served up with a trailing slash, it'd be a different story.